src/Chapter1.hs

```diff
@@ -615,7 +625,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastDigit n + secondLast n
+      where
+        secondLast :: Int -> Int
+        secondLast n = (`div` 10) $ (abs n) `mod` 100 - lastDigit n
```

Comment:
Nice work on reusing already defined `lastDigit`! Though, I see that the implementation for `secondLast` can be improved a bit. How about, if you do `div n 10` and then just `mod result 10`?

Reply:
That makes a lot more sense!